lib/genesis/test_utils.go

```diff
@@ -100,12 +101,33 @@ func CreateTestGenesisJSONFile(t *testing.T, asRaw bool) (filename string) {
 	return filename
 }
 
-// NewTestGenesisWithTrieAndHeader generates genesis, genesis trie and genesis header
-func NewTestGenesisWithTrieAndHeader(t *testing.T) (*Genesis, *trie.Trie, *types.Header) {
+func getAbsolutePath(t *testing.T, p string) string {
```

Comment:
maybe rename `p` I'm not sure what it is from the function signature

I guess `pathFromRepoRoot`?